### PR TITLE
A few improvements to logs

### PR DESCRIPTION
**Pull request by davidiw**
* NodeConfigs are printed on startup and that includes the whole genesis blob that can be massive and unreadable. This eliminates that from being printed.
* Simplify startup of node by merging execution-helpers and removing the random node config if one isn't produced

### CODE REVIEW

**Comment by ankushagarwal** [comment]
Looks good to me. Will let someone else stamp.

File: common/executable-helpers/src/helpers.rs [hunk 1]
```diff
@@ -1,64 +0,0 @@
...
-use std::path::Path;
-
-pub fn load_config_from_path(config_path: Option<&Path>) -> NodeConfig {
-    // Load the config
-    let node_config = match config_path {
-        Some(path) => {
-            info!("Loading node config from: {}", path.display());
-            NodeConfig::load(path).expect("Failed to load node config.")
-        }
-        None => {
```
Comment (davidiw) [inline common/executable-helpers/src/helpers.rs]: bye bye

**Pull request closed without merging**